tests/integrationv2/providers.py

```diff
@@ -231,8 +245,13 @@ class OpenSSL(Provider):
     _version = get_flag(S2N_PROVIDER_VERSION)
 
     def __init__(self, options: ProviderOptions):
-        self.ready_to_send_input_marker = None
         Provider.__init__(self, options)
+        # We print some OpenSSL logging that includes stderr
+        self.expect_stderr = True
+
+    @classmethod
+    def get_send_marker(cls):
+        return 'Verify return code'
```

Comment:
`get_send_marker` is an arbitrary string right? in that case it might be nice to include the lib we are testing: `'Testing OpenSSL'`.

'Verify return code' implies to me that we are only testing that feature, which is not true i think.

Reply:
Unfortunately it's not arbitrary.

The send marker is some output that a provider like OpenSSL / S2N prints when it's ready to begin receiving STDIN. For example, OpenSSL apparently prints "Verify return code" once it has performed the handshake and is waiting for data to send across the connection. See how it's used: https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/processes.py#L172-L183